CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
```

Comment:
very nit: I find that ["project"](https://dictionary.cambridge.org/dictionary/english/project) term is not properly used. I propose to be more precise 😉 

Moreover, I do not understand why not using Present Simple tense.

```suggestion
All notable changes to OpenTelemetry Go Automatic Instrumentation
is documented in this file.
```

PS. I am aware that this copied from https://keepachangelog.com/en/1.1.0/ 😉

Reply:
```suggestion
All notable changes to OpenTelemetry Go Automatic Instrumentation are documented in this file.
```